src/main/java/com/revature/ams/AirportRunner.java

```diff
@@ -25,7 +38,12 @@ public static void main(String[] args) {
             System.out.println("1. Flight Information"); // ctrl + d on a line will repeat
             System.out.println("2. Add Flights");
             System.out.println("3. Update Flight"); // sout - shorthand for System.out.println();
-            System.out.println("4. Exit"); // sout - shorthand for System.out.println();
+            // TODO: REVIEW ME
+            System.out.println("4. Register");
```

Comment:
To make it clear, you can say Register Member in line 42.

Reply:
Always a good idea for our intentions with each actions to be clearer for our users.